File: JavaScript_Courses/JavaScript_Basics/Preparation/05_While_Loop_Lab/05_Account_Balance.js
function accountBalance(input) {
    let balance = 0;

    i = 0;
    while (true) {
        let currentCommand = input[i];
        i++;

        if (currentCommand == "NoMoreMoney") {
            break;
        }

        currentAmount = Number(currentCommand);
        if (currentAmount < 0) {
            console.log('Invalid operation!');
            break;
        }

        balance += currentAmount;
        console.log(`Increase: ${currentAmount.toFixed(2)}`);
    }

    console.log(`Total: ${balance.toFixed(2)}`);
}


// accountBalance(["5.51",
//     "69.42",
//     "100",
//     "NoMoreMoney"]);

File: Python_Courses/Python_Basics/Preparation/05_While_Loop_Lab/05_Account_Balance.py
current_balance = 0

command = input()
while not command == "NoMoreMoney":
    current_sum = float(command)
    if current_sum < 0:
        print("Invalid operation!")
        break

    current_balance += current_sum
    print(f"Increase: {current_sum :.2f}")
    command = input()

print(f"Total: {current_balance :.2f}")
